packages/scaffolding-cli/templates/src/ssr/components/Header/__snapshots__/index.test.tsx.snap

```diff
@@ -44,6 +44,7 @@ exports[`renders Header snapshot 1`] = `
           Yumido
         </a>
       </h2>
+      0
```

Comment:
What is this being rendered?

Reply:
I think it is the create button missing. Will look into it